src/Intl/Transliterator/Transliterator.php

```diff
@@ -0,0 +1,761 @@
+        'bg' => 'Bulgarian-Latin',
+        'bn' => 'Bengali-Latin',
+        'el' => 'Greek-Latin',
+        'fa' => 'Persian-Latin',
+        'he' => 'Hebrew-Latin',
+        'hy' => 'Armenian-Latin',
+        'ka' => 'Georgian-Latin',
+        'kk' => 'Kazakh-Latin',
+        'ky' => 'Kirghiz-Latin',
+        'ko' => 'Korean-Latin',
+        'mk' => 'Macedonian-Latin',
+        'mn' => 'Mongolian-Latin',
+        'or' => 'Oriya-Latin',
+        'ps' => 'Pashto-Latin',
+        'ru' => 'Russian-Latin',
+        'sr' => 'Serbian-Latin',
+        'th' => 'Thai-Latin',
+        'tk' => 'Turkmen-Latin',
+        'uk' => 'Ukrainian-Latin',
+        'uz' => 'Uzbek-Latin',
+        'zh' => 'Han-Latin',
+    );
+
+    /**
+     * @var array|null
+     */
+    private static $TRANSLIT;
+
+    /**
+     * @var array<string, array<string, string>>|null
+     */
+    private static $ASCII_MAPS;
+
+    /**
+     * @var array<string, int>|null
+     */
+    private static $ORD;
+
+    /**
+     * @var string
+     */
+    private static $ASCII = "\x20\x65\x69\x61\x73\x6E\x74\x72\x6F\x6C\x75\x64\x5D\x5B\x63\x6D\x70\x27\x0A\x67\x7C\x68\x76\x2E\x66\x62\x2C\x3A\x3D\x2D\x71\x31\x30\x43\x32\x2A\x79\x78\x29\x28\x4C\x39\x41\x53\x2F\x50\x22\x45\x6A\x4D\x49\x6B\x33\x3E\x35\x54\x3C\x44\x34\x7D\x42\x7B\x38\x46\x77\x52\x36\x37\x55\x47\x4E\x3B\x4A\x7A\x56\x23\x48\x4F\x57\x5F\x26\x21\x4B\x3F\x58\x51\x25\x59\x5C\x09\x5A\x2B\x7E\x5E\x24\x40\x60\x7F\x00\x01\x02\x03\x04\x05\x06\x07\x08\x0B\x0C\x0D\x0E\x0F\x10\x11\x12\x13\x14\x15\x16\x17\x18\x19\x1A\x1B\x1C\x1D\x1E\x1F";
+
+    /**
+     * @var string|null
+     */
+    private $id;
+
+    /**
+     * @var int
+     */
+    private $direction = \Transliterator::FORWARD;
+
+    private function __construct()
+    {
+    }
+
+    public function __get($name)
+    {
+        if ('id' === $name) {
+            return $this->id;
+        }
+
+        return null;
+    }
+
+    public function __isset($name)
+    {
+        if ('id' === $name) {
+            return isset($this->id);
+        }
+
+        return false;
+    }
+
+    public function __set($name, $value)
+    {
+        if ('id' === $name) {
+            trigger_error('The property "id" is read-only', E_USER_WARNING);
+
+            return;
+        }
+
+        if ('id_intern' === $name) {
+            $this->id = $value;
+        }
+    }
+
+    /**
+     * Returns an replacement array for ASCII methods with one language.
+     *
+     * For example, German will map 'ä' to 'ae', while other languages
+     * will simply return e.g. 'a'.
+     *
+     * @psalm-suppress InvalidNullableReturnType - we use the prepare* methods here, so we don't get NULL here
+     *
+     * @param string $language [optional] <p>Language of the source string e.g.: en, de_at, or de-ch.
+     *                         (default is 'en') | ASCII::*_LANGUAGE_CODE</p>
+     *
+     * @return array{orig: string[], replace: string[]}
+     *                     <p>An array of replacements.</p>
```

Comment:
why using HTML tags ?

Reply:
Good question. I added them because I saw them in the PhpStorm stubs and you can easily add something like <strong> ... but anyway, I have removed all html tags with one of my last commit here.